cogs/member_stats.py

```diff
@@ -1,21 +1,82 @@
 import discord
-from discord.ext import commands
+from discord.ext import tasks, commands
 from pathlib import Path
 from .validation import is_moderator, is_mod_commands_channel
 import json
+import os
+from os import path
+from datetime import datetime
+from dateutil import tz
 
 
 class MemberStats(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.status = 0
```

Comment:
Perhaps we could change `self.status` to something like `self.isRecording`, and then use boolean values. What do you think? Might make things a bit more clearer. Then later, we can make things a bit more self-documenting with
```py
if self.isRecording:
    ..
```